runtime/src/main/java/com/vaadin/quarkus/QuarkusInstantiator.java

```diff
@@ -15,39 +15,19 @@
  */
 package com.vaadin.quarkus;
 
-import jakarta.enterprise.context.ApplicationScoped;
-import jakarta.enterprise.inject.spi.BeanManager;
-import jakarta.inject.Inject;
 import java.util.concurrent.atomic.AtomicBoolean;
 import java.util.stream.Stream;
 
-import io.quarkus.arc.Unremovable;
+import jakarta.enterprise.inject.spi.BeanManager;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import com.vaadin.flow.component.Component;
 import com.vaadin.flow.di.DefaultInstantiator;
 import com.vaadin.flow.di.Instantiator;
 import com.vaadin.flow.i18n.I18NProvider;
-import com.vaadin.flow.server.VaadinService;
 import com.vaadin.flow.server.VaadinServiceInitListener;
-import com.vaadin.quarkus.annotation.VaadinServiceEnabled;
 
-/**
- * Instantiator implementation based on Quarkus DI feature.
- *
- * Quarkus DI solution (also called ArC) is based on the Contexts and Dependency
- * Injection for Java 2.0 specification, but it is not a full CDI
- * implementation. Only a subset of the CDI features is implemented.
- *
- * See <a href="https://quarkus.io/guides/cdi-reference">Quarkus CDI
- * Reference</a> for further details.
- *
- * @see Instantiator
- */
```

Comment:
Perhaps we can preserve Javadocs, since the class is public

Reply:
Not sure on that, the docs states that this Instantiator is using DI feature - after introducing InstantiatorFactory it is a plain java class. Factory is DI bean now. Indeed it is public but user needs to overwrite factory first to then use custom Instantiator. I've simplified the docs for this class.

Reply:
My understanding of the original docs is that `Instantiator` is **internally** using DI feature (e.g.`BeanManager`)